example/app/api/views.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import viewsets
+
+from .. import models
+from . import serializers
+
+
+class CRUDView(
+    viewsets.ModelViewSet,
+):
```

Comment:
```suggestion
class CRUDView(viewsets.ModelViewSet):
```

And maybe change name to `ModelWithFilesViewSet)` to be consistent?